README.md

```diff
@@ -1,15 +1,62 @@
-# MoonDeck ![Status](https://github.com/FrogTheFrog/moondeck/actions/workflows/build.yaml/badge.svg) [![Chat](https://img.shields.io/badge/Chat-on%20discord-7289da.svg)](https://discord.com/invite/U88fbeHyzt) ![Decky store](https://img.shields.io/badge/dynamic/json?color=blue&label=Decky%20version&query=%24%5B%3F%28%40.name%3D%3D%27MoonDeck%27%29%5D.versions%5B0%5D.name&url=https%3A%2F%2Fplugins.deckbrew.xyz%2Fplugins)
+# ***MOONDECK*** ![Status](https://github.com/FrogTheFrog/moondeck/actions/workflows/build.yaml/badge.svg) [![Chat](https://img.shields.io/badge/Chat-on%20discord-7289da.svg)](https://discord.com/invite/U88fbeHyzt) ![Decky store](https://img.shields.io/badge/dynamic/json?color=blue&label=Decky%20version&query=%24%5B%3F%28%40.name%3D%3D%27MoonDeck%27%29%5D.versions%5B0%5D.name&url=https%3A%2F%2Fplugins.deckbrew.xyz%2Fplugins)
 
 A plugin that lets you play any of your Steam games via Moonlight without needing to add them to Sunshine first, providing a similar experience to GeForce GameStream or Steam Remote Play.
 
 ![quicksettings](.github/assets/quickmenu.png)
 
 ## What is it really?
 
-MoonDeck is an automation tool that will simplify launching your Steam games via the Moonlight client for streaming.
+***MOONDECK*** is an automation tool that will simplify launching your Steam games via the Moonlight client for streaming. It simplifies the launch of a specific game to a single click, similar to how Steam's Remote Play feature works.
 
 It requires an additional lightweight app to be installed on the host PC - [MoonDeck Buddy](https://github.com/FrogTheFrog/moondeck-buddy). Additional one-time setup instructions can be found within the settings page of the plugin itself.
 
+## Why would I need it?
+***Game streaming via Moonlight, by DEFAULT, follows these steps:***
+
+1. Open moonlight as an application on the steam deck
+2. Select a Host machine to stream from
+3. Select an application to stream, usually steam Big Picture Mode (BPM)
+5. Select a game via BPM interface 
+6. Selected Game is launched
+
+
+***Game Streaming via MOONDECK follows these steps:***
+1. Press ***MOONDECK*** button on game to stream within SteamDeck's OS 
+2. Selected Game is launched via moonlight stream
+
+![applicationLaunch](.github/assets/moondeck_launch_button.png)
+
+***MOONDECK*** sets configurations ahead of time and skipping many of the steps each time a stream is started. This allows a streamlined experience of selecting a game directly from the steam deck's OS and immediately starting the game stream.
+
+
+**More importantly, the ***Moonlight, by DEFAULT,*** method caused SteamDeck's OS to ONLY see *Moonlight* as the running application, rather than *any specific game*.**
```

Comment:
I find it very hard to read when it's italic and bold, especially when italic is mixed withing bold style sentences.

Please avoid bold sentences and use only bold to empathize something (not bold+italic). For example:
```
**MOONDECK** sets configurations ahead of time and skipping many of the steps each time a stream is started.
```
```
More importantly, the **Moonlight, by DEFAULT,** method caused SteamDeck's OS to ONLY see **Moonlight** as the running application, rather than **any specific game**.
```